modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,340 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met.
+
+.Prerequisites
+
+* A multi-AZ ROSA cluster deployed with STS and created using customer-managed Subnets
+* Access to modify the VPC and Subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} (ROSA) CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone, the Local Zone is enabled for the account and the ALB Operator is available within the Local Zone/region.
+====
+
+
+.Procedure
+
+. Specify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS using the following commands:
```

Comment:
Specify is an odd word for this. Here, customer is identifying these properties. Perhaps, we use 'Identify'?

Reply:
> Specify is an odd word for this. Here, customer is identifying these properties. Perhaps, we use 'Identify'?

Ack. Changed. (sidenote: I was using "Specify based on an existing doc in our doc set, but I think "Identify is better". Will see how it holds up going through review process.).